proposed/swift-interop.md

```diff
@@ -106,6 +106,18 @@ Rejected Option 5 would have provided a better ".NET" shape than our selected al
 
 In the SwiftAsync calling convention, there is also an Async Context register, similar to the Self and Error registers. Like the error register, the async context must be passed by a pointer value. As a result, similar options apply here, with the same constraints. Additionally, we don't already have an existing `CallConvAsync` calling convention modifier, so going the calling convention route like proposed for the self register is not practical. As a result, we will likely need to use a special type like `SwiftAsyncContext` to represent the async context register, similar to the proposals for the error register and the self register.
 
+##### Value types marked as @frozen
```

Comment:
I think this is covered in https://github.com/dotnet/designs/blob/3768dc57e7d90f2b21e17464bffcb590b157a810/proposed/swift-interop.md#structsenums paragraph already.

Can we use the plan described in that paragraph? (Handle non-frozen structs in projection layer, and assume that all structs are frozen in the low-level calling convention.)